examples/whisper/code/user_script.py

```diff
@@ -10,7 +10,7 @@
 
 
 def get_encoder_decoder_init():
-    model = WhisperForConditionalGeneration.from_pretrained("openai/whisper-tiny")
+    model = WhisperForConditionalGeneration.from_pretrained("<model_name>")
```

Comment:
is it possible that pass the model_name as parameter of get_encoder_decoder_init? it seems the model name already in the config file

Reply:
I intentionally didn't want to do that when I was working on custom components stuff. But it might be helpful cause custom components may need model name. I will add it.